fineract-provider/src/main/java/org/apache/fineract/commands/service/CommandWrapperBuilder.java

```diff
@@ -2736,4 +2736,27 @@ public CommandWrapperBuilder deleteSelfServiceBeneficiaryTPT(final Long benefici
         return this;
 	}
 
-}
\ No newline at end of file
+	public CommandWrapperBuilder createReportMailingJob(final String entityName) {
+        this.actionName = "CREATE";
+        this.entityName = entityName;
+        this.entityId = null;
+        this.href = "/reportmailingjobs";
+        return this;
+    }
+    
+    public CommandWrapperBuilder updateReportMailingJob(final String entityName, final Long entityId) {
```

Comment:
Caller is passing permission name and entity id. But you are assigning permission name to entity name